contracts/TransakMulticallExecuter.sol

```diff
@@ -34,15 +42,23 @@ contract TransakMulticallExecuter is
      */
     function multiCall(
         address[] calldata targets,
-        bytes[] calldata data
-    ) external onlyOwner nonReentrant returns (bytes[] memory) {
+        bytes[] calldata data,
+        uint256[] calldata value
+    ) external payable onlyOwner nonReentrant returns (bytes[] memory) {
+        require(
+            reduce(value) >= msg.value,
```

Comment:
@shivamangina Shouldn't the `msg.value` be greater then `reduce(value)` here?

Reply:
Yes, Fixed